crates/uv/src/commands/project/init.rs

```diff
@@ -549,6 +550,10 @@ impl InitProjectKind {
             )?;
         }
 
+        // Create a `py.typed` file
+        let py_typed = src_dir.join("py.typed");
+        fs_err::write(py_typed, "")?;
```

Comment:
Maybe this should be within the `if !init_py.try_exists()?`... Otherwise, we're creating `src/py.typed` even if the source is contained elsewhere in an existing project.

Reply:
The code isn't super clear because of that variable name, but it's actually `src/<module>/py.typed` and `src/<module>/__init__.py`. I think this is reasonable to create even if the `__init__.py` already exists since we're (in theory) initializing the metadata to publish the library.

Reply:
Any possibility that this might clobber a file that might already exist?

Reply:
Yep 🤦‍♀️ https://github.com/astral-sh/uv/pull/7338